pages/index.js

```diff
@@ -21,27 +22,84 @@ export default function HomePage() {
     .filter((transaction) => transaction.type === "expense")
     .reduce((acc, transaction) => acc - transaction.amount, 0);
   const balanceTotal = incomeTotal - expenseTotal;
-  console.log(balanceTotal);
 
   return (
     <>
-     <StyledAccountBalance balance={balanceTotal}>
+      <StyledAccountBalance balance={balanceTotal}>
         <p>Account balance</p>
         {balanceTotal}
       </StyledAccountBalance>
-    
-     
+
+      <TransactionsList>
+        {transactions.map((transaction) => {
+          return (
+            <TransactionItem key={transaction._id}>
+              {new Intl.NumberFormat("de-DE", {
+                style: "currency",
+                currency: "EUR",
+              }).format(Number(transaction.amount) || 0)}
+              <Amount $type={transaction.type}>
+                {transaction.type && `${transaction.type}`}
+              </Amount>
+              <CategoryBadge>
+                {transaction.category ?? "Uncategorisiert"}
+              </CategoryBadge>
+              <DateText>
+                {new Date(transaction.date).toLocaleDateString("de-DE")}
+              </DateText>
+            </TransactionItem>
```

Comment:
please create a distinct Component for the TransactionItem in Order to hide some of the complexity and make the code more readable.